class.mesh.php

```diff
@@ -447,6 +447,29 @@ function save_post( $post_id, $post ) {
 				update_post_meta( $section->ID, '_mesh_template', $template );
 			}
 
+			/**
+			 * Process Section Meta
+			 */
+
+			$default_section_meta = array(
+				'css_class',
+				'lp_equal',
+				'title_display',
+				'push_pull',
+				'collapse',
+				'blocks',
+				'post_title',
+				'post_status',
+				'template',
+				'menu_order'
+			);
+
+			/**
+			 * This filter is used to remove or add elements to the default section meta
+			 * @todo "meta" related to a section
+			 */
+			apply_filters( 'mesh_default_section_meta_fields', $default_section_meta );
```

Comment:
Apply filters needs to return it's value. https://developer.wordpress.org/reference/functions/apply_filters/

`$default_section_meta = apply_filters( 'mesh_default_section_meta_fields', $default_section_meta );`